agent/php_user_instrument.c

```diff
@@ -291,7 +295,9 @@ static void nr_php_wrap_user_function_internal(nruserfn_t* wraprec TSRMLS_DC) {
 }
 
 static nruserfn_t* nr_php_user_wraprec_create(void) {
-  return (nruserfn_t*)nr_zalloc(sizeof(nruserfn_t));
+  nruserfn_t* wr = (nruserfn_t*)nr_zalloc(sizeof(nruserfn_t));
+  wr->pid = nr_getpid();
```

Comment:
check `wr` for NULL.

Reply:
Checking that `wr` is not `NULL` is redundant - it is guaranteed by `nr_zalloc` that its return value is not `NULL`. See https://github.com/newrelic/newrelic-php-agent/blob/R11.7/axiom/util_memory.c#L57-L61.